src/e2e/app/cli/repo-tests/src/commands/test_login_command.rs

```diff
@@ -124,3 +125,79 @@ pub async fn test_login_logout_oauth(kamu_node_api_client: KamuApiServerClient)
 }
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub async fn test_login_add_repo(kamu: KamuCliPuppet) {
+    let dummy_access_token = "dummy-access-token";
+    let dummy_url = "http://example.com";
+
+    // Login with skipping adding repo
+    kamu.assert_success_command_execution(
+        [
+            "login",
+            dummy_url,
+            "--access-token",
+            dummy_access_token,
+            "--skip-add-repo",
+        ],
+        None,
+        Some([format!("Login successful: {dummy_url}").as_str()]),
+    )
+    .await;
+
+    let repo_list = kamu.get_list_of_repos().await;
+    assert!(repo_list.is_empty());
```

Comment:
Optional: `pretty_assertions::assert_eq!([], repo_list)` may be a bit more convenient when debugging, since it will immediately display repositories, if available